src/main/java/com/robohorse/robopojogenerator/generator/postprocessors/AbsPostProcessor.java

```diff
@@ -10,69 +10,87 @@
 import com.robohorse.robopojogenerator.models.GenerationModel;
 
 import javax.inject.Inject;
+
 import java.util.Set;
 
 /**
  * Created by vadim on 23.10.16.
  */
 public abstract class AbsPostProcessor {
+
     @Inject
-    ClassGenerateHelper generateHelper;
+    ClassGenerateHelper    generateHelper;
     @Inject
     ClassTemplateProcessor classTemplateProcessor;
 
+
     public String proceed(ClassItem classItem, GenerationModel generationModel) {
+
```

Comment:
Oh, that is my own coding style.

Add the empty line will make code look better.

If you don't like it, just reformat it.